plugins.d/Lets_Encrypt/dehydrated-wrapper

```diff
@@ -132,8 +141,8 @@ stop_server() {
 
 restart_servers() {
     for servicename in $@; do
-        info "starting $servicename"
-        service $servicename restart | tee -a $LOG
+        info "(Re)starting $servicename"
+        systemctl restart $servicename | tee -a $LOG
```

Comment:
Should we maybe prefer to still use service as it still works inside a chroot?

Reply:
FWIW, the reason why I changed it is that the Tomcat9 package only has a `tomcat9.service` file (i.e. a systemd service config file) and I'm not completely clear how that would work with the `service` command (I assume it would work, but I haven't tested). Besides, I'm not sure if we need to support getting Let's Encrypt certs within a build chroot... So thanks for bringing it up and asking the question, but I'm going to leave it as is.